README.md

```diff
@@ -298,6 +298,8 @@ Filters are methods on an existing pipe that also return a pipe, allowing you to
 | [`Filter`](https://pkg.go.dev/github.com/bitfield/script#Pipe.Filter) | user-supplied function filtering a reader to a writer |
 | [`FilterLine`](https://pkg.go.dev/github.com/bitfield/script#Pipe.FilterLine) | user-supplied function filtering each line to a string|
 | [`FilterScan`](https://pkg.go.dev/github.com/bitfield/script#Pipe.FilterScan) | user-supplied function filtering each line to a writer |
+| [`DecodeBase64`](https://pkg.go.dev/github.com/bitfield/script#Pipe.DecodeBase64) | decode base64 encoded input |
```

Comment:
Can we keep these in alphabetical order as well? It's an easy thing to overlook, but a list that's _mostly_ in order is very confusing for readers.